src/js/popupButtons.js

```diff
@@ -0,0 +1,83 @@
+function popupButtons(id) {
+  let watchedMoviesIds = [];
+  let queueMoviesIds = [];
+  const buttonQueue = document.querySelector('.queue');
+  const buttonWatched = document.querySelector('.watched');
+  const buttons = document.querySelector('.button');
+
+  if (localStorage.getItem('queueMoviesIds')) {
```

Comment:
please, create cinemaService and move all the logic with getting and setting the localStorage data to that service

Reply:
i did